XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,66 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.CornerRadiusPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:AppUIBasics"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+    <StackPanel>
+        <TextBlock IsTextSelectionEnabled="True">
+            <LineBreak />
+            <Run Text="For basic UI elements like buttons that users see on screen at all times," />
+            <Run Text="the default radius value is" />
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for is" />
+            <Run FontWeight="Bold" Text="4px" />
```

Comment:
Maybe we can have a sample or information on the themeresource that is used for popups and flyouts (which is called OverlayCornerRadius)?

Reply:
How do I set an overlay cornerRadius?

Reply:
You would need to set the `OverlayCornerRadius` resource in your app.xaml file for it to take effect (at least that is the usual way to achieve that).

Reply:
Okay